### ISSUE TITLE
Progressbar icon change request

### ISSUE THREAD

**Issue by adventuretrail** [nmap/nmap#2324]
Can we change the color of the progress bar? It is difficult to distinguish whether it is running or not as both shades look the same.
Some high contract shades can help, currently many times while using remotely due to low FPS or privacy screen filter etc. it becomes difficult to identify scan status. 
![progressbar](https://user-images.githubusercontent.com/77768357/122510022-7d06df80-d022-11eb-8e60-1c8c6976a410.png)

**Comment by fyodor** [comment]
Good point.  If someone wants to create (or find), test, and submit a new one (with no copyright problems), I'm all for it.  The goal of the throbber is to show the user that the scan is running without putting a fake progress bar which might suggest more than Zenmap really knows about what percentage of the whole scan is done.  Nmap itself just reports what phase is in and roughly what percentage OF THAT PHASE of that particular target group is done.

**Comment by goarblt** [comment]
hello
https://www.evil.com/
